algorithms/Python/searching/Breadth-first search.py

```diff
@@ -0,0 +1,32 @@
+# this graph to check the algorithm

```

Comment:
the algorithm better fit `graph` section than `searching` 

also according to the [contribution guidelines](https://github.com/MakeContributions/DSA#contribution-guidelines) of the repo, each word in file name should be separated by a hyphen "-" character instead of whitespace, so a recommended name of the file should be `Breadth-first-search.py`

Reply:
this technique for searching on graphs or tree

Reply:
I am sorry but now I will change the name file and upload it again?

Reply:
just change the path of the file, I shouldn't be an issue ( both on Github interface or via git CLI )

check [this blog](https://docs.github.com/en/enterprise-server@3.1/repositories/working-with-files/managing-files/moving-a-file-to-a-new-location) for details

Reply:
> this technique for searching on graphs or tree

up to you then.